build/source/engine/stomResist.f90

```diff
@@ -385,11 +359,10 @@ subroutine stomResist_flex(&
  character(*),intent(out)        :: message                    ! error message
  ! ------------------------------------------------------------------------------------------------------------------------------------------------------
  ! general local variables
- character(LEN=256)              :: cmessage                   ! error message of downwind routine
  logical(lgt),parameter          :: testDerivs=.false.         ! flag to test the derivatives
  real(dp)                        :: unitConv                   ! unit conversion factor (mol m-3, convert m s-1 --> mol H20 m-2 s-1)
  real(dp)                        :: rlb                        ! leaf boundary layer rersistance (umol-1 m2 s)
- real(dp)                        :: x0,x1,x2,x3,x4             ! temporary variables
+ real(dp)                        :: x0,x1,x2                   ! temporary variables
```

Comment:
Perhaps we need to revise the coding conventions.

Here x0,x1,x2 have different meaning, and just used to simplify the code (e.g., see lines 532 and 536).